src/pages/net-neutrality/petition.js

```diff
@@ -9,26 +9,44 @@ import SignupForm from '../../components/net-neutrality/signup-form.js';
 import FormBody from '../../components/signup-form/form-body.js';
 import Modal from '../../components/modal.js';
 
+const DONATE_CTA_DELAY = 1000; // in milliseconds
+
 var Signup = React.createClass({
   getInitialState: function() {
     return {
       showModal: false,
+      dismissedModal: false,
       signupSuccess: false
     };
   },
   componentDidMount: function() {
-    /*if (!this.props.location.query.subscribed) {
-      setTimeout(() => {
-        this.setState({
-          showModal: true
-        });
-      }, 3000);
-    }*/
+    if (typeof window !== "undefined" && window.document) {
```

Comment:
what's this check for?

Reply:
I had onscroll on document initially, so this should become window && window.addEventListener